src/datanode/src/tests/instance_test.rs

```diff
@@ -185,67 +185,6 @@ async fn test_execute_insert() {
     assert!(matches!(output, Output::AffectedRows(2)));
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn test_execute_insert_by_select() {
```

Comment:
This test is moved to sqlness case.

Reply:
I think we should keep this test, it's part of the integration test. The sqlness test is a black box test and the developer may not run it while developing.